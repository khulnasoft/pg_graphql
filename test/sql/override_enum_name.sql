begin;
    create type account_priority as enum ('high', 'standard');
    comment on type public.account_priority is E'@graphql({"name": "CustomerValue"})';

    select jsonb_pretty(
        graphql.resolve($$
        {
          __type(name: "CustomerValue") {
            enumValues {
              name
            }
          }
        }
        $$)
    );

rollback;
