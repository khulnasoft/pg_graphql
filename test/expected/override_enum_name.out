begin;
    create type account_priority as enum ('high', 'standard');
    comment on type public.account_priority is E'@graphql({"name": "CustomerValue"})';
    select jsonb_pretty(
        graphql.resolve($$
        {
          __type(name: "CustomerValue") {
            enumValues {
              name
            }
          }
        }
        $$)
    );
              jsonb_pretty              
----------------------------------------
 {                                     +
     "data": {                         +
         "__type": {                   +
             "enumValues": [           +
                 {                     +
                     "name": "high"    +
                 },                    +
                 {                     +
                     "name": "standard"+
                 }                     +
             ]                         +
         }                             +
     }                                 +
 }
(1 row)

rollback;
